lte/gateway/release/build-magma.sh

```diff
@@ -24,7 +24,7 @@ VERSION=1.7.0 # magma version number
 SCTPD_MIN_VERSION=1.7.0 # earliest version of sctpd with which this version is compatible
 
 # RelWithDebInfo or Debug
-BUILD_TYPE=Debug
+BUILD_TYPE=RelWithDebInfo
```

Comment:
why is this file modified? Looks like you need rebasing?

Reply:
> @pmishra-wavelabs please do not forget to set the product label for 5g sa when you raise PRs. I added the label for this one.

Sure